### PR TITLE
ReAdd peer meta. 

### PR DESCRIPTION
**Pull request by qiuyesuifeng**

### CODE REVIEW

**Comment by qiuyesuifeng** [comment]
@tiancaiamao @siddontang @disksing  PTAL

File: server/cluster_worker.go [hunk 1]
```diff
@@ -295,8 +299,8 @@ func (c *raftCluster) handleAddPeerReq(region *metapb.Region) (uint64, error) {
 		storeID := store.GetId()
 
 		existStore := false
-		for _, peerStoreID := range region.StoreIds {
-			if peerStoreID == storeID {
+		for _, peer := range region.Peers {
+			if peer.GetStoreId() == storeID {
 				// we can't add peer in the same store.
```
Comment (tiancaiamao) [inline server/cluster_worker.go]: tip: maybe we'll allow this later, but allocate a new peer id and recycle the old peer.

Comment (siddontang) [inline server/cluster_worker.go]: why should support this? 
A store can only have one peer for the region. Pd must remove first, then add again.

**Comment by disksing** [comment]
LGTM

**Comment by tiancaiamao** [comment]
LGTM

**Pull request merged**